tsconfig.json

```diff
@@ -11,6 +11,12 @@
     "allowJs": true
   },
   "exclude": ["node_modules"],
-  "include": ["src/**/*", "docs/swagger.ts", "test", "docs"],
+  "include": [
+    "src/**/*",
+    "docs/swagger.ts",
+    "test",
+    "docs",
+    "src/middleware/permissionCheck"
```

Comment:
```suggestion

```

I think you can safely remove this because no such file exists in the code base.